torch/_inductor/scheduler.py

```diff
@@ -428,6 +428,7 @@ def decide_inplace_update(self) -> None:
                         )
                         and buffer_reuse_key(input_buf.node)
                         == buffer_reuse_key(buf.node)
+                        and buf.get_name() not in V.graph.removed_buffers
```

Comment:
This condition should be hoisted out of the `for read in ordered_reads` loop but otherwise this LGTM

Reply:
Thanks for the suggestions and changed. Please kindly help to take a look again.